src/coreclr/src/debug/createdump/createdump.cpp

```diff
@@ -12,13 +12,13 @@ bool g_diagnostics = false;
 bool
 CreateDump(const char* dumpPath, int pid, MINIDUMP_TYPE minidumpType)
 {
-    ReleaseHolder<DumpDataTarget> dataTarget = new DumpDataTarget(pid);
-    ReleaseHolder<CrashInfo> crashInfo = new CrashInfo(pid, dataTarget, false);
+    ReleaseHolder<CrashInfo> crashInfo = new CrashInfo(pid);
```

Comment:
I guess that is historical and not necessary anymore.

Reply:
I left CrashInfo allocated because it is a IUnknown and I want the ReleaseHolder to clean it up. I did change DumpWriter to be stack allocated and moved DumpDataTarget to EnumerateMemoryRegionsWithDAC.